src/commands.ts

```diff
@@ -694,7 +722,18 @@ export const insertSection =
 
     let pos
     if (hasParentNodeOfType(schema.nodes.body)(selection) || subsection) {
-      pos = findPosAfterParentSection(state.selection.$from)
+      /*
+      Determine the position where the new section or subsection will be inserted.
```

Comment:
Thanks for adding a comment. While this is very much appreciated it's not a very opaque feature and a relatively small piece of code. So we can probably do with a more laconic commentary. I'd say in this case a coupe of lines should be plenty.

Reply:
OK...